src/app.js

```diff
@@ -29,24 +34,63 @@ app.use(
 
 app.use(express.static("public"));
 app.use(express.urlencoded({ extended: true }));
+app.use(setGa4ContainerId);
+app.use(setStatusCode);
+
+const routeInfo = [
```

Comment:
Usually we don't store an array of data in the app.js. It's the starting point of a node app and needs to stay clean.
+ when you want to store objects like that, it's better to define an id for every object. That will let you find it easily if you need extract one of them.

Reply:
Might be a good one the constants file?